code/datums/perks/cooldown.dm

```diff
@@ -0,0 +1,50 @@
+/datum/perk/cooldown
+	var/perk_lifetime = 3 MINUTES
+	var/timestamp_start
+	gain_text = "You feel tired. Your body needs some time to recover from all this training."
+
+/datum/perk/cooldown/assign(mob/living/carbon/human/H)
+	if(..())
+		timestamp_start = world.time
+
+/datum/perk/cooldown/on_process()
+	if(!..() || ((timestamp_start + perk_lifetime) > world.time))
+		remove()
+
+/datum/perk/cooldown/exertion
+	name = "Overexertion"
+	desc = "Your muscles hurt after an intense workout. \
+			Your TOUGHNESS stat is reduced for some time. \
```

Comment:
```suggestion
			Your TGH stat is reduced for some time. \
```

Reply:
I think they should be referred either as "Toughness stat" or "TGH stat" or just "TGH"/"Toughness" for consistency